src/MathText.tsx

```diff
@@ -42,7 +42,8 @@ export type MathTextProps = Pick<MathTextRowProps, 'direction' | 'containerStyle
 
 export const InlineMathItem = React.memo(({ value, isMath, CellRendererComponent, Component, inline, style, ...props }: MathTextItemProps) => {
     if (value === '') return null;
-    const config = useMemo(() => ({ inline }), [inline]);
+    const { config: configProps } = props as MathTextItemProps<true>;
+    const config = useMemo(() => ({ ...(configProps || {}), inline }), [inline, configProps]);
```

Comment:
If config is always passed as optional then you should change `MathTextItemProps` it seems.
`config?: MathViewProps['config']`



https://github.com/ShaMan123/react-native-math-view/blob/a089f98a28d278a6a52e8ed6891e114fff50e588/src/common.tsx#L22C22-L22C37

Reply:
Is that correct?

Reply:
The `MathTextItemProps` extends from `MathViewCommonProps` which defines `config` as optional, which I guess is intentional? Because if you look at the `MathTextItemProps` it optionally extends from `MathViewCommonProps` if the generic type is a boolean. Not sure what the purpose is for this optional extension.

```export type MathTextItemProps<T extends boolean = boolean> = (T extends true ? Omit<MathViewProps, 'math'> : TextProps) & {
    value: string,
    isMath: T,
    Component?: MathView,
    CellRendererComponent?: ElementOrRenderer,
    inline?: boolean
}```

Reply:
So I prefer to spread `config: configProp` in line 43 if that is possible.
Sorry for the delay and for not being acquainted with the code anymore. 

Actually, I think it is best you fork and I archive this repo with a pinned issue stating so so other devs can write there what they want.